jekyll/_docs/configuration.md

```diff
@@ -512,9 +510,9 @@ deployment:
       - ./deploy_master.sh
 ```
 
-***Note*** The `deployment` section doesn't support sub-sections such as `pre` 
-that you might find elsewhere in `circle.yml`. If you get a strange error such 
-as "commands must be a list" when in the `deployment` section, this is likely 
+***Note*** The `deployment` section doesn't support sub-sections such as `pre`
+that you might find elsewhere in `circle.yml`. If you get a strange error such
+as "commands must be a list" when in the `deployment` section, this is likely
```

Comment:
@smart-alek thanks for the suggestion! created a topic on discuss here: https://discuss.circleci.com/t/schema-linter-for-circle-yml-files/8817

cc @mohit